Resources/doc/reference/introduction.rst

```diff
@@ -73,6 +73,9 @@ By default the ``SonataPageBundle`` is shipped with core block services:
 
  - sonata.page.block.container      : Block container
  - sonata.page.block.children_pages : Render a navigation panel
+ - sonata.page.block.breadcrumb     : Render a breadcrumb
```

Comment:
I didn't say your addition is useless. I said your addition is **not related** to your PR/commit.

cc @greg0ire